ui/src/utils/groupBy.js

```diff
@@ -92,11 +99,15 @@ const constructCells = serieses => {
         vals,
```

Comment:
Not relevant to this pr but maybe something to consider: instead of `values || []` here, you can give values on line 91 a default value of [].